src/app/models/genes.ts

```diff
@@ -98,6 +99,8 @@ export interface Gene {
   input_data_display_value?: string;
 
   bio_domains?: BioDomains;
+
+  ensembl_info: EnsemblInfo[];
```

Comment:
Is `ensembl_info` an array of `EnsemblInfo` in the database? Just curious, since the ticket seems to specify that `ensembl_info` is just an object of `EnsemblInfo`

Reply:
It is an array in MongoDB but that is a bug.  This will be fixed in a future data release and will require some code changes to match.

Reply:
Got it! Would it be useful to add a FIXME comment here with the associated ticket number, so it's easier to track the fix in the future?

Reply:
Added!